src/app/models/evt-models.ts

```diff
@@ -127,6 +127,28 @@ export interface WitnessGroup {
     groupId: string;
 }
 
+export type ApparatusEntriesList = Map<ApparatusEntry>;
```

Comment:
Just use 'Map<ApparatusEntry>', also because it is not a List

Reply:
?